cmd/parser/parser.go

```diff
@@ -7,8 +7,8 @@ import (
 	"log"
 	"os"
 	"path/filepath"
-	"strings"
 
+	"github.com/ghodss/yaml"
```

Comment:
We're using `"gopkg.in/yaml.v2"` in another section of code - can we stick to one of those two?

Reply:
This package extends the one you mentioned, but you're right it would be a good idea to stick to one.

Reply:
Done.